source/globus/local_endpoints.rst

```diff
@@ -0,0 +1,11 @@
+Users can create and manage endpoints on their personal computers via `Globus Connect Personal <https://www.globus.org/globus-connect-personal>`_. Globus Connect Personal is available for all major operating systems. Download the software and follow the installation instructions. During the setup process you will be asked to set the local endpoint. Each endpoint has a unique ID. Those ones created or managed by you are associated with your Globus account.
```

Comment:
Since this link would be used in multiple files, I'd suggest to put in a `links.rst` file that can be included at the bottom each each file you want to use the link in.